File: backend/src/main/java/br/com/calciolari/datahub/imports/infrastructure/interpdv/qrp/EmfTextRun.java
package br.com.calciolari.datahub.imports.infrastructure.interpdv.qrp;

/**
 * Text run extracted from {@code EMR_EXTTEXTOUTW} (type 84), as in the PoC
 * {@code parseEmfTexts}.
 */
public record EmfTextRun(int pageIndex, int x, int y, String text) {
}

File: backend/src/Calciolari.DataHub/Imports/Infrastructure/InterPdv/Qrp/EmfTextRun.cs
namespace Calciolari.DataHub.Imports.Infrastructure.InterPdv.Qrp;

/// <summary>
/// Text run extracted from EMR_EXTTEXTOUTW (type 84), as in the PoC parseEmfTexts.
/// </summary>
public sealed record EmfTextRun(int PageIndex, int X, int Y, string Text);
